src/game/text.c

```diff
@@ -336,6 +363,8 @@ static void Text_DrawText(TEXTSTRING *textstring)
         int32_t sprite_num = letter;
         if (letter >= 16) {
             sprite_num = m_TextASCIIMap[letter - 32];
+        } else if (letter == '\t') {
+            sprite_num = RIGHT_ARROW_SYM;
```

Comment:
This comment is out of scope for this PR but I don't like this approach at all. This define exists only because we ran out of letters, doesn't it? That way it only perpetuates the already terrible character table hacking from Core.
```
"Ammo: %d {icon_sg_ammo}"
```
Related: #386

Reply:
Ya I couldn't find another character to use. And these changes are all just internal text changes to prepare for the arrows in the menu options, so they are in a way related.

Reply:
Yes I ran out of characters. And these changes are all internal text changes to support arrows so they are related in a way.